docs/commandline.rst

```diff
@@ -1,92 +1,358 @@
 Command Line Usage
 ==================
 
-This chapter describes how to use radish from the command line. All it's commands, options and arguments.
+This chapter describes how to use Radish from the command line. All it's
+commands, options and arguments.
+
 
 Run - Specify Feature files
 ---------------------------
 
-All arguments which doesn't belong to any command line switch are interpreted as Feature files or Feature file locations. If the argument is a directory all files ending with ``.feature`` will be run. It's possible to mix files and directories:
+All arguments which doesn't belong to any command line switch are interpreted
+as Feature files or Feature file locations. If the argument is a directory all
+files ending with ``.feature`` will be run. It's possible to mix files and
+directories:
 
 .. code:: bash
 
   radish SomeFeature.feature myfeatures/
 
+
 Run - Specify base directory
 ----------------------------
 
-The directory where the *Step* and *Terrain* files are located is called the ``base directory``.
-Per default it points to ``$PWD/radish``. All python files within this directory are imported by radish. You can specify the base directory with the ``-b`` or ``--basedir`` switch:
+Radish looks for *Step* and *Terrain* files in the ``base directory``. By
+default the ``base directory`` is set to the in the ``radish`` folder within
+the current working directory (a.k.a ``$PWD/radish``). Radish imports all
+the python files imported.
```

Comment:
Remove redundant `imported` in the end of the sentence and reword sentence a little